### PR TITLE
fix: base url

### PR DESCRIPTION
**Pull request by climba03003**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by climba03003** [comment]
image
Interesting, it is how the preview works.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -14,6 +14,10 @@ on:
   # daily schedule
   workflow_call:
 
+# TODO: remove once it's production
+env:
+  BASE_URL: /website-next/
```
Comment (Eomm) [inline .github/workflows/publish.yml:19]: It has been 10 years that I have wanted to learn this setup:

image


I think it is the right time 😄

**Comment by Eomm** [comment]
> I am not sure if the below is strictly needed.

It is, otherwise that folder will be not included in the build

**Comment by climba03003** [comment]
> It is, otherwise that folder will be not included in the build

`static` is copied in the top level of `build`.

I pretty sure the map of route is `/website-next/` mapped to the top-level of project which means `/website-next` folder should never been access.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (HassanBahati) [reply .github/workflows/publish.yml:19]: go for it:rocket: :rocket: can imagine the high levels of energy with which you'll be setting it up:fire::fire:

**Comment by HassanBahati** [comment]
> > I am not sure if the below is strictly needed.
> 
> It is, otherwise that folder will be not included in the build

And perhaps is the reason why currently the site is failing to deploy properly...

**Comment by HassanBahati** [comment]
> The reason is missing the `base url` which generate the wrong url. Not because of missing folder of `/website-next`

True

**Review by Eomm — COMMENTED** [review]
Let's try to merge it

**Comment by climba03003** [comment]
Interesting, the deployment isn't upload to any branch.
It is actually using environment and displaying the artifact.

**Comment by Eomm** [comment]
I think the files are here:

image


But IDK if it will be deleted after 1 week of inactivity tho

I think this repo/project is interesting to learn this side of GH

**Pull request merged**